POSTER_SESSIONS/README.md

```diff
@@ -1,53 +1,101 @@
+| 10 | The intricacies of DWeb Addressing | The challenges with the origin policy and all details to have in consideration when adding IPFS natively to Web Browsers resolution. Consider base58 -> base32 migrations, case sensitivity, browser vendors, etc. |
+| 11 | KeyStore & Linked Data Key (aka Multikey) | Self describing keys. What kind of formats are there? How do we handle them all? What do we use them for? |
+| 12 | The design of the IPFS Repo | What is the IPFS Repo? How does it store data? How can users store data in multiple backends (local fs, IndexedDB, S3 and so on) |
+| 13 | DEX - The Importers and Exporters Project | Getting and formatting data into and out of IPFS. |
+| 14 | IPFS/libp2p PubSub | Publisher-Subsciber message model. What implementations exist for this (floodsub, gossipsub)? How do they compare? What are uses of it? How does it interact with (our) other dynamic data systems? |
+| 15 | NPM on IPFS | Bring up left-pad ;^) |
+| 16 | InterPlanetary Test Lab | |
+| 17 | CAR |  Content addressable archive(s). How does it work? What are some applications of it? |
+| 18 | SECIO | Encryption method/protocol. How does it compare to alternatives (like TLS)? |
+| 19 | Rendezvous Protocol | Peer discovery method. |
+| 20 | Circuit Relay | Message relaying. What scenarios is this useful in / what problems does this solve? |
+| 21 | DNSLink | Utilize DNS to point to IPFS data. |
+| 22 | xTP | External Transport Protocol. How do IPFS nodes communicate with foreign protocols? Why would we need this? |
+| 23 | Go-ds-crdt - Merkle CRDTs |  |
+| 24 | DAG walking facilities in IPLD |  |
+| 25 | Core API | Higher level API for interacting with IPFS nodes. |
+| 26 | New Go HTTP API libs  | Lower level node API. |
+| 27 | QUIC | Transport protocol. When and why is it useful to use QUIC over alternatives? What is the importance of good transports in general? |
+| 28 | CRDTs in Peerpad |  |
+| 29 | Stream Muxers | Libp2p mplex, yamux, SPDY etc. |
+| 30 | Websocket Transport | Why Websockets exist? Why TCP and QUIC can be used everywhere? |
+| 31 | Performance | Where and why can things be slow in a typical IPFS publish or request operation? |
+| 32 | CPU | What are the most compute-heavy operations in a production IPFS node and why? |
+| 33 | IPNS | Mutable references. How does it interact with the keystore? How do things get resolved? When to use it over other dynamic systems? |
+| 34 | Routing | How does IPFS handle routing? (how does IPFS abstract peers via IDs instead of location?) What discovery options exist? (kad-dht, rendezvous). How do we take advantage of multiple discovery methods? |
+| 35 | Graphsync | IPLD graph synchronizing. How does it relate to other systems (bitswap, IPLD)?  What benefit does it give us? |
+| 36 | HTTP gateways | Interop bridge between HTTP <-> IPFS. How do I make one of these? Why would I want one? |
```

Comment:
This reminds me on having one about js-ipfs Infrastructure, related to:

- https://github.com/ipfs/js-ipfs/issues/1459
- https://github.com/ipfs/js-ipfs/issues/2093

@alanshaw it could be immensely useful for JS IPFS users.

Reply:
Yes I'd be up for that!

Reply:
@djdv wanna add it to the list? :)

Reply:
adding as
```
js-ipfs Infrastructure
 - How do peers connect?
 - What limitations are imposed by the environment (browsers, node, etc.)?
```